content/engineering/our-approach/architecture-reviews/data-act-pilot.md

```diff
@@ -1,160 +1,71 @@
 ---
-title: 'DATA Act Pilot: Simplicity is Key'
+title: 'DATA Act Pilot: Simplicity is key'
 sidenav: true
 sticky_sidenav: true
 permalink: /engineering/data-act-pilot/
```

Comment:
Ah, good catch. Are there a lot of url renames like this? I'm not sure what's best. Should we keep these the same as the existing guides just for now, then come back to it post-migration?

Reply:
I am not sure what is the best path here. It has come up as part of a different [issue](https://github.com/18F/TLC-crew/issues/338), and I saw again that it's mentioned in [Anita's presentation ](https://docs.google.com/presentation/d/1c6W2zBvlUV8L-hgMqIAhXXmuhfzh5teP100rvHsur00/edit#slide=id.g275d934f5af_0_16), but I am don't know if we came to a decision. Let's check with Alex when he is back, because I think he may have a clearer picture from the different conversations.